### DISCUSSION TITLE
Vert.x gateway configuration

### DISCUSSION THREAD

**Discussion by shachindrasingh** [apiman/apiman#2119] [Help] [answered]
Hi ,
i am getting the error while doing the vert.x gateway configuration . i had follow all step that mention in document still getting the same error(below)
"Authentication to the gateway failed.  Perhaps check that your credentials are correct."

if any one have document pls share the same

**Comment by volkflo** [comment]
Can you show your configuration from "Adminstration" -> "Manage Gateway" -> "The Gateway"?

**Reply by shachindrasingh** [reply]
End Point : http://localhost:8081/apiman-gateway-api/
apimanager/apiman123!

**Comment by volkflo** [comment, marked as answer]
For the vertx gateway you can remove the /apiman-gateway-api
End Point : http://localhost:8081 should be fine then

**Reply by shachindrasingh** [reply]
Still the same issue while testing the gateway .

Authentication to the gateway failed.  Perhaps check that your credentials are correct.

[ERROR] 2022-06-21 13:21:01.352 [vert.x-eventloop-thread-1] KeycloakOAuthFactory - Access Token Error: Not Found: {"error":"RESTEASY003210: Could not find resource for full path: http://localhost:8080/auth//realms/apiman/protocol/openid-connect/token"}.

**Reply by volkflo** [reply]
Did you checked, that the credentials are correct?
There is also a `//`in the url auth//relams. Please check your configuration carefully again and make sure that secrets, pubic key etc. are correct

**Reply by shachindrasingh** [reply]
Thanks @volkflo  issue resolved..

**Reply by msavy** [reply]
@shachindrasingh is there an issue with the default configuration? If so, would be good to know so we can fix in the upstream before we do `3.0.0.Final`?

**Reply by shachindrasingh** [reply]
@msavy that is token issue. in default configuration realm-public-key/secret need to replace rest remain same.  also please update the document with the apiman ui gateway configuration . 
Also can we disable the keycloak auth ?
